stdlib/FileWatching/src/pidfile.jl

```diff
@@ -280,7 +280,7 @@ function open_exclusive(path::String;
 end
 
 function _rand_filename(len::Int=4) # modified from Base.Libc
-    slug = Base.StringMemory(len)
+    slug = Base.StringVector(len)
```

Comment:
Overall, looks good to me. I wonder why this here uses `StringVector` over `StringMemory`? As far as I can see, the memory can never escape the function.

Reply:
Since this is not performance-sensitive I wanted to avoid using the internal function here.